tests/suites/test_suite_pk.function

```diff
@@ -432,10 +432,6 @@ void invalid_parameters( )
                             mbedtls_pk_parse_key( &pk,
                                                   NULL, sizeof( buf ),
                                                   buf, sizeof( buf ) ) );
-    TEST_INVALID_PARAM_RET( MBEDTLS_ERR_PK_BAD_INPUT_DATA,
-                            mbedtls_pk_parse_key( &pk,
-                                                  buf, sizeof( buf ),
-                                                  NULL, sizeof( buf ) ) );
```

Comment:
Yes:
```
    TEST_ASSERT( mbedtls_pk_parse_key( &pk, NULL, 0, NULL, 1 ) ==
                 MBEDTLS_ERR_PK_KEY_INVALID_FORMAT );
```
I removed it precisely because it is valid and that test failed.

Reply:
OK